src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
```

Comment:
Instead of having a chunk of code in main, you could put the code in functions and call the functions from main. Have a look at week 3 lecture on single level of abstraction per function (SLAP).

Reply:
Thank you! will edit it later